test/server/rules.bzl

```diff
@@ -15,23 +15,24 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-def grakn_java_test(name, grakn_artifact_linux, grakn_artifact_mac, deps = [], classpath_resources = [], data = [], **kwargs):
+def grakn_java_test(name, native_grakn_artifact, deps = [], classpath_resources = [], data = [], **kwargs):
+    native.java_test(
+        name = name,
+        deps = depset(deps + ["@graknlabs_common//test/server:grakn-setup"]).to_list(),
+        classpath_resources = depset(classpath_resources + ["@graknlabs_common//test/server:logback"]).to_list(),
+        data = depset(data + [native_grakn_artifact]).to_list(),
+        args = ["$(location " + native_grakn_artifact + ")"],
+        **kwargs
+    )
 
+def native_grakn_artifact(name, mac_artifact, linux_artifact, **kwargs):
```

Comment:
Essentially apart from some minor renames, the only change is that this is no longer nested inside `def grakn_java_test`.